.github/CONTRIBUTING.md

```diff
@@ -58,3 +59,111 @@ Things we pay attention in a PR :
 
 
 _If you are unsure about git you can have a look on our [git tips to have a clean history](https://github.com/mockito/mockito/wiki/Using git to prepare your PR to have a clean history)._
+
+
+## Coding style
+
+This section is not about some kind of fruitless tabs vs spaces debate. It's about having the code readable, the project grows and it is not rare to read contributions from many different individuals. Each one of us has a different writing style, we are fine with this. Without enforcing we may be picky about it, however this we think this may improve helps readability for everyone.
```

Comment:
> however this we think this may improve helps readability for everyone.

Seems some words were incorrectly placed? `however we think that this improves the readibility of the code for everyone`.